learn.html

```diff
@@ -106,16 +106,25 @@
                 <h1>Work with data from a subsystem</h1>
                     <p>Data from the Net Zero house includes 400+ variables that include readings from instrumentation around the house in one minute increments. Each of these variables, also called channels in NIST documentation, provides the readings from one instrument, and the instruments are grouped into subsystems:
                     <br>
-                    <br>
-                    <i>DHW - Domestic Hot Water</i> <i>SHW - Solar Hot Water</i> <i>Load - Electrical and Thermal Loads by equipment and people</i> <i>HVAC - Heating and Cooling System</i> <i>IndEnv - Indoor Environment</i> <i>PV - Photovoltaic</i> <i>OutEnv - Outdoor Environment</i> <i>Vent - Ventilation</i> <i>Elec - Electrical</i> 
+                    <ul>
+                    <li>DHW - Domestic Hot Water</li> 
+                    <li>SHW - Solar Hot Water</li> 
+                    <li>Load - Electrical and Thermal Loads by equipment and people</li> 
+                    <li>HVAC - Heating and Cooling System</li> 
+                    <li>IndEnv - Indoor Environment</li> 
+                    <li>PV - Photovoltaic</li> 
+                    <li>OutEnv - Outdoor Environment</li> 
+                    <li>Vent - Ventilation</li> 
+                    <li>Elec - Electrical</li> 
+                    </ul>
                     </p>
-
-                    <p> This tutorial walks through loading data from a particular subsystem.</p>
+                    <br>
+                    <p>This tutorial walks through loading data from a particular subsystem.</p>
  
         <div id="Data-From-Subsystem-A" class="subgroup">
 
                 <h3>Get the data you need</h3>
-                <p>If you want to analyze Net-Zero data for specific subsystems, rather than data for the whole house, you can download the data from the Net-Zero website and import it into R, or you can use the download link directly.</p>
+                <p>If you want to analyze net-zero data for specific subsystems, rather than data for the whole house, you can download the data from the Net Zero website and import it into R, or you can use the download link directly.</p>
```

Comment:
Is there a reason that you lowercased 'Net-Zero'? I'm just looking for consistency b/c Net-Zero isn't lowercased on data.html or on line 107. Also, should we be using a hyphen?